superset/views/core.py

```diff
@@ -888,21 +736,6 @@ def explore(
             except DatasetNotFoundError:
                 pass
         datasource_name = datasource.name if datasource else _("[Missing Dataset]")
-
-        if datasource:
-            if config["ENABLE_ACCESS_REQUEST"] and (
-                not security_manager.can_access_datasource(datasource)
-            ):
-                flash(
-                    __(security_manager.get_datasource_access_error_msg(datasource)),
-                    "danger",
-                )
-                return redirect(
```

Comment:
Note this redirect no longer worked as this is being called  via a `GET` yet the redirect URL only accepts a `POST`.